aws/index.ts

```diff
@@ -190,14 +144,18 @@ if (config.installAwsOTelCollector) {
     awsOTelCollector,
     {
       metadata: {
-        namespace: namespaceName,
+        namespace: config.OpenTelemetry.Collector.Namespace,
         labels: awsOTelCollectorLabels,
         name: "aws-otel-config.yaml",
       },
       data: { [podConfig]: awsOTelConfig.readAwsOTelCollectorConfig() },
     },
-    { provider: cluster.k8sProvider },
+    {
+      provider: cluster.k8sProvider,
+      dependsOn: [otelCollectorNamespace],
```

Comment:
I've got some flaky failures because of that. Therefore, it is better to be clear about such dependencies.